tests/tests_models.py

```diff
@@ -1,5 +1,7 @@
 # -*- coding: utf-8 -*-
 from django.test import TestCase
+from django.core.management import call_command
+from django.utils.six import StringIO
 from filer.models import File as FilerFile
```

Comment:
Please add a blank line between Django and filer imports. Probably isort is not configured properly in this project.